docs/en/latest/xrpc/redis.md

```diff
@@ -0,0 +1,102 @@
+:::
+
+## Granularity of the request
+
+Like other protocols based on the xRPC framework, the Redis implementation here also has the concept of `request`.
+
+Each Redis command is considered a request. However, the message subscribed from the server won't be considered a request.
+
+For example, when a Redis client subscribes to channel "foo" and receives the message "bar", then it unsubscribes the "foo" channel, there are two requests: `subscribe foo` and `unsubscribe foo`.
+
+## Attributes
+
+| Name | Type          | Required | Default                                       | Valid values                                                       | Description                                                                                                                                                                                                                                           |
+|----------------------------------------------|---------------|----------|-----------------------------------------------|--------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| faults | array[object]        | False    |                                               |  | Fault injections which can be applied based on the commands and keys |
+
+Fields under an entry of `faults`:
+
+| Name | Type          | Required | Default                                       | Valid values                                                       | Description                                                                                                                                                                                                                                           |
+|----------------------------------------------|---------------|----------|-----------------------------------------------|--------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| commands | array[string]        | True    |                                               | ["get", "mget"]  | Commands fault is restricted to |
+| key | string        | False    |                                               | "blahblah"  | Key fault is restricted to |
+| delay | number        | True    |                                               | 0.1  | Duration of the delay in seconds |
+
+## Example usage
+
+Assumed the APISIX is proxying TCP on port 9101, and the Redis is listening on port 6379.
```

Comment:
```suggestion
Assumed the APISIX is proxying TCP on port `9101`, and the Redis is listening on port `6379`.
```

Reply:
Why quote a simple number?

Reply:
For the reader's attention, the port should be quoted. We need to be consistent with the other documentation. https://apisix.apache.org/docs/apisix/next/getting-started#before-you-begin